src/test/java/org/sagebionetworks/bridge/sdk/integration/SubpopulationTest.java

```diff
@@ -18,18 +18,23 @@
 import org.sagebionetworks.bridge.sdk.models.ResourceList;
 import org.sagebionetworks.bridge.sdk.models.holders.GuidVersionHolder;
 import org.sagebionetworks.bridge.sdk.models.subpopulations.Subpopulation;
+import org.sagebionetworks.bridge.sdk.models.subpopulations.SubpopulationGuid;
 
 public class SubpopulationTest {
 
+    private TestUser admin;
     private TestUser developer;
+    private SubpopulationGuid guid;
     
     @Before
     public void before() {
+        admin = TestUserHelper.getSignedInAdmin();
         developer = TestUserHelper.createAndSignInUser(SubpopulationTest.class, false, Roles.DEVELOPER);
     }
     
     @After
     public void after() {
+        admin.getSession().getAdminClient().deleteSubpopulationPermanently(guid);
```

Comment:
Should have a null check on guid before deleting the SubPop. Otherwise, if creating the Subpop fails, the After will likely also fail with a NullPointerException and mask the original error.